packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -1287,7 +1287,12 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
       const operationMap = this.getOperationMap(columns, aggregations);
       const operationOrder = this.getOperationOrder(aggregations);
 
-      return { operationMap, operationOrder, showOnTop };
+      return {
+        operationMap,
+        operationOrder,
+        showOnTop,
+        defaultOperation: AggregationOperation.SKIP,
```

Comment:
How does enterprise handle this? Does it just ignore it for now?

Reply:
Handles it as it should. The `SKIP` operation was already implemented on Enterprise side, and it was also already looking for `defaultOperation` in the totals table config: https://github.com/deephaven-ent/iris/blob/304a3267f77cff5afedc96cf004f4352b59dba68/web/client-api/src/main/java/com/illumon/iris/web/client/api/JsTotalsTableConfig.java#L69

Tested against dev-grizzly with these changes pulled in using `start-community`.